src/components/components/TotalRaisedEtherCountup.js

```diff
@@ -15,7 +15,7 @@ export default ({ totalRaised }) => {
     start: decimals,
     end: decimals,
     delay: 1000,
-    duration: 6,
+    duration: 1,
```

Comment:
Very interesting, I see now that it counts up almost every second. Is this since there are many more animals, each animals contribution is added individually and counted up. Maybe in the future when we have way more animals, we will need to group the update and make the ticker a bit less chaotic.

Reply:
It counts up every 2 seconds, and it counts up (goes green) for 1 second. We can change this.

All the counters will count up at the same time, (including on the info screen for "remaining deposit" and amount collected by the animal.